Backend/Linkedout/GraphQL/Queries/UserQueries.cs

```diff
@@ -19,7 +19,7 @@ Lazy<IMediator> mediator
             _mediator = mediator;
         }
 
-        [Authorize]
```

Comment:
Is there a reason for removing it?

Reply:
Just for testing and forget to put it again